CGPs/cgp-0080.md

```diff
@@ -0,0 +1,130 @@
+---
+cgp: '0080'
+title: 'Follow-on Funding for Climate Collective'
+date-created: '2023-04-25'
+author: 'Anna Lerner, Nirvaan Ranganathan, Ed Walters'
+status: 'DRAFT'
+discussions-to: 'https://forum.celo.org/t/climate-collective-1-year-anniversary-heres-to-many-more/5585'
+governance-proposal-id: '104 (TBD)'
+date-executed: 'TBD'
+---
+
+# Abstract
+
+After achieving the key goals of our initial governance proposal [(CGP-52)](https://celo.stake.id/#/proposal/60) through Climate Collective’s grants program established in 2022, we request follow-on funding from the Celo community in order to catalyze the growth of Celo’s climate ecosystem in 2023 and 2024.
+
+## Background
+Climate Collective (CC) recently published a [“Year 1 Update & Preview of Upcoming CGPs”](https://forum.celo.org/t/climate-collective-1-year-anniversary-heres-to-many-more/5585) forum post that contextualizes this proposal, and is advisable to review before proceeding. More than 44 individuals representing over 30 organizations in the Celo community responded with resoundingly positive validation, including testimonials on our past work and useful suggestions for this next phase of CC’s growth. Key outcomes from our initial proposal are detailed in the Appendix section.
+
```

Comment:
Its worth noting that at this time the governance contract is overallocated by previous initiatives by ~600K CELO. There is the Mento proposal to give back a large amount but it is currently only a signalling proposal and it may be good to mention something about it in this proposal or wait until that is cleared to submit this.

Reply:
Thanks for flagging @0xzoz - my understanding from the Mento proposal (https://celo.stake.id/#/proposal/102) is that "After execution of the proposal, the Mento Reserve is instructed to send CELO to the on-chain Community Fund". Hence I removed the reference to Mento here as we plan to submit this CGP-80 after Mento's CGP-79 is executed on Saturday 4/29.

Reply:
Thanks for clarifying. Just wanted to make sure it was all lined up and accounted for. Without the extra funds in the Governance contract having spend approvals greater that the existing balance(which is the case now) creates a dynamic between the initiatives that may not have been considered in the original economics assumptions.

Reply:
Indeed and thanks for checking @0xzoz . Could we merge this PR today ahead of the governance call tomorrow 4/28?